docs/cql-queries.md

```diff
@@ -11,7 +11,7 @@ If you like to use the command line, please look into [this section](cql-queries
 
 ## API Documentation
 
-If yopu like to use the CQL Evaluation API directly, please read the [CQL API Documentation](cql-queries/api.md).
+If you like to use the CQL Evaluation API directly, please read the [CQL API Documentation](cql-queries/api.md).
```

Comment:
```suggestion
If you'd like to use the CQL Evaluation API directly, please read the [CQL API Documentation](cql-queries/api.md).
```

Reply:
Thanks. I commit that change by myself, because I have changed some other minor things in between.